src/query.rs

```diff
@@ -101,6 +103,12 @@ impl Serialize for Asset {
         let mut state = serializer.serialize_struct("Asset", 2)?;
         state.serialize_field("asset_id", &format!("{}", &self.asset_id))?;
         state.serialize_field("asset_quantity", &self.asset_quantity)?;
+        let raw_metadata = serialize(&self.metadata);
+        //raw_metadata contains data length (VarInt). The length must be removed to serialize for json.
+        let deserialized: Vec<u8> =
+            deserialize(&raw_metadata[..]).expect("faild to parse metadata");
+        let hex = hex::encode(deserialized);
+        state.serialize_field("metadata", &hex)?;
```

Comment:
Is it useful to use UTF8 string instead of hex?

Reply:
Yes, utf8 string may be useful.
But I don't know how the response should be represented, in case of binary metadata.
Do you have any idea?

Reply:
For example, 

utf8 string, 
```
"metadata": {
    "utf8": "u=https://cpr.sm/5YgSU1Pg-q",
    "hex":  "753d68747470733a2f2f6370722e736d2f35596753553150672d71"
}
```

, and binary

```
"metadata": {
    "hex":  "0102"
}
```

Reply:
oh, I see.

Reply:
I implemented serialization for metadta in openassets-tapyrus.

https://github.com/chaintope/openassets-tapyrus/pull/3

Reply:
I see. It seems to get an error.